lfs/transfer_queue.go

```diff
@@ -125,11 +127,17 @@ func (q *TransferQueue) processBatch() {
 
 	objects, err := Batch(transfers)
 	if err != nil {
-		q.errorc <- err
-		sendApiEvent(apiEventFail)
-		return
+		if isNotImplError(err) {
+			tracerx.Printf("queue: batch not implemented, disabling")
+			configFile := filepath.Join(LocalWorkingDir, ".gitconfig")
+			git.Config.UnsetLocalKey(configFile, "lfs.batch")
```

Comment:
I think that's good. People can always set it in `~/.gitconfig` if they really want.

Though this makes me realize that the config applies to every remote. Eventually we may want also support keys like `remote.{name}.lfsbatch` so that a repository can happily use the batch API.

Considering that the batch API will likely be the ONLY API eventually, remote-specific config keys is probably too much hassle.